code/core/src/test/java/com/adobe/marketing/mobile/services/ui/notification/PushTemplateTypeTest.kt

```diff
@@ -0,0 +1,62 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.services.ui.notification
+
+import org.junit.Test
+import kotlin.test.assertEquals
+
+class PushTemplateTypeTest {
+
+    @Test
+    fun `test fromString returns BASIC value`() {
```

Comment:
Given that we have more types being added, how do you feel about simplifying this to 2 tests that do not need updating for newer types.

```
// positive test for fromString for each of the enum values
        PushTemplateType.values().forEach { value ->
            assertEquals(value, PushTemplateType.fromString(value.value))
        }
```

```
// negative test to ensure that invalid values map to unknown
assertEquals(PushTemplateType.UNKNOWN, PushTemplateType.fromString("SomeRandomValue"))
```

Reply:
Sorry I accidentally merged it. Would it be ok if @PratthamArora addressed this in a subsequent PR?

Reply:
Thanks @prudrabhat for the suggestion, will make the changes and open another PR